src/utils/constants.js

```diff
@@ -0,0 +1,10 @@
+const Settings = {
```

Comment:
@arizzitano I know we've gone back and forth on this some more, but I was thinking, Settings still seems too broad for me (but am fine keeping it if we still think it makes the most sense) but what if we called these Patterns? I remember hearing UI more recently use "Pattern Library" as terminology even if it's not the same meaning as the deprecated Pattern Library we had?